http/README.md

```diff
@@ -8,11 +8,12 @@ The service is configured using the environment variables presented in the
 following table. Note that any unset variables will be replaced with their
 default values.
 
-| Variable             | Description         | Default               |
-|----------------------|---------------------|-----------------------|
-| MF_HTTP_ADAPTER_PORT | Service HTTP port   | 8180                  |
-| MF_NATS_URL          | NATS instance URL   | nats://localhost:4222 |
-| MF_THINGS_URL        | Things service URL  | localhost:8181        |
+| Variable                    | Description                    | Default               |
+|-----------------------------|--------------------------------|-----------------------|
+| MF_HTTP_ADAPTER_LOG_LEVEL   | Log level for the HTTP Adapter | error                 |
+| MF_HTTP_ADAPTER_PORT        | Service HTTP port              | 8180                  |
+| MF_NATS_URL                 | NATS instance URL              | nats://localhost:4222 |
+| MF_THINGS_URL               | Things service URL             | localhost:8181        |
```

Comment:
I think that there is a code block explaining how to run a service just under these tables. Should look something like:
```
MF_THINGS_URL=[Things service URL] MF_NATS_URL=[NATS instance URL] MF_HTTP_ADAPTER_PORT=[Service HTTP port] $GOBIN/mainflux-http
```
Please, add this env variable there, as well. Please do this in the other README.md files, as well.

Reply:
Ah yep, I missed this, I will get that cleaned up!